kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -25,20 +26,28 @@ import {
 import { PageNames } from './constants';
 
 function facilityParamRequiredGuard(toRoute, subtopicName) {
+  const { isNavigationFailure, NavigationFailureType } = VueRouter;
   if (store.getters.userIsMultiFacilityAdmin && !toRoute.params.facility_id) {
-    router.replace({
-      name: 'ALL_FACILITIES_PAGE',
-      params: { subtopicName },
-    });
-    return true;
+    router
+      .replace({
+        name: 'ALL_FACILITIES_PAGE',
+        query: { subtopicName },
```

Comment:
In case the above comment is unclear, this should be `params` in place of `query`.

Reply:
No, I addressed it .